examples/advanced/swarm_learning/pt/cifar10_data_utils.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Are these code under "pt" the same as the centralized CIFAR10 example? If so, we should consolidate to reuse. This will also make it clear that all existing training code can be used without changes.